packages/core/parcel/src/cli.js

```diff
@@ -60,6 +60,7 @@ const commonOptions = {
   '--cache-dir <path>': 'set the cache directory. defaults to ".parcel-cache"',
   '--no-source-maps': 'disable sourcemaps',
   '--no-content-hash': 'disable content hashing',
+  '--unsafe-inlining': 'enabled potentionally unsafe inlining operations',
```

Comment:
hmm, how should we decide when to include an option in the CLI vs only in target config?